src/core/components/card/metrics-card.scss

```diff
@@ -61,6 +61,10 @@
   & > p {
     margin: 0;
   }
+  background-color: #c2d4d4;
```

Comment:
We should be using color tokens and spacing  from carbon
e.g 
```scss
margin-botton: layout.$spacing-05;
color: $color-gray-70;
```

the above are just example of how it should be done.